test/x3/char_set.cpp

```diff
@@ -0,0 +1,89 @@
+#define BOOST_SPIRIT_X3_UNICODE
```

Comment:
You'll need a copyright notice at the top with the Boost license. See other files.

Reply:
Added!